NikoBracket/InfoViewController.m

```diff
@@ -25,52 +29,77 @@ - (void)viewDidLoad {
     self.infoTableView.dataSource = self;
     self.infoTableView.rowHeight = 125;
     
+    [self fetchData];
+    
     //Initialize UIRefreshControl
     self.refreshControl = [[UIRefreshControl alloc] init];
-    [self.refreshControl addTarget:self action:@selector(beginRefresh:) forControlEvents:UIControlEventValueChanged];
+    [self.refreshControl addTarget:self action:@selector(fetchData) forControlEvents:UIControlEventValueChanged];
     [self.infoTableView insertSubview:self.refreshControl atIndex:0];
-
+    
+    
 }
 
-- (void)beginRefresh:(UIRefreshControl *)refreshControl {
+- (void)fetchData{
+    // parse teams json file
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"teams" ofType:@"json"];
+    NSData *data = [NSData dataWithContentsOfFile:path];
+    self.teams = [[NSMutableDictionary alloc] init];
+    NSArray *temp = [NSJSONSerialization JSONObjectWithData:data options:kNilOptions error:nil];
+    for (int i = 0; i < temp.count; i++) {
+        self.teams[temp[i][@"Key"]] = temp[i];
+    }
+    
+    //parse 1st round matchups from json
+    NSError *error;
+    NSString *url_string = [NSString stringWithFormat: @"https://api.sportsdata.io/v3/cbb/scores/json/Tournament/2022?key=003d3b7e4d724cbeb785ca7d4aded519"];
```

Comment:
It might also be good to put the API endpoint string as a constant at the top of the file or class, something like `const NSString kMatchUpsEndpoint = @"https://api.sportsdata.io/v3/cbb/scores/json/Tournament/2022?key=";`